mkdocs.yml

```
@@ -4,6 +4,14 @@ site_name: User Documentation
 repo_url: https://github.com/ezsystems/user-documentation
 site_url: https://doc.ibexa.co/projects/userguide/en/latest/
 copyright: "Copyright 1999-2024 Ibexa AS and others"
+validation:
+    # omitted_files: warn # TODO: Decide on persona_paths/icons.md
```

persona_paths/icons.md removal in #350

```suggestion
    omitted_files: warn
```